### PR TITLE
Update docs for tf.contrib.layers.optimize_loss  …

### PR DESCRIPTION
**Pull request by schaber**
Easier to read and corrects name of example callable

### CODE REVIEW

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet, @martinwicke: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

File: tensorflow/contrib/layers/python/layers/optimizers.py [hunk 1]
```diff
@@ -109,11 +109,12 @@ class should be sub-class of `tf.Optimizer` that implements
...
                           variables will be multiplied by given constant.
-    clip_gradients: float, callable or `None`. If float, is provided, a global
-      clipping is applied to prevent the norm of the gradient to exceed this
-      value. Alternatively, a callable can be provided e.g.: adaptive_clipping.
-      This callable takes a `list` of `(gradients, variables)` `tuple`s and
-      returns the same thing with the gradients modified.
+    clip_gradients: float, callable or `None`. If a float is provided, a global
+      clipping is applied to prevent the norm of the gradient from exceeding
+      this value. Alternatively, a callable can be provided, e.g.,
+      adaptive_clipping_fn().  This callable takes a `list` of
```
Comment (fchollet) [inline tensorflow/contrib/layers/python/layers/optimizers.py]: Please use backquotes around code keywords.

File: tensorflow/contrib/layers/python/layers/optimizers.py [hunk 2]
```diff
@@ -109,11 +109,12 @@ class should be sub-class of `tf.Optimizer` that implements
...
-    clip_gradients: float, callable or `None`. If float, is provided, a global
-      clipping is applied to prevent the norm of the gradient to exceed this
-      value. Alternatively, a callable can be provided e.g.: adaptive_clipping.
-      This callable takes a `list` of `(gradients, variables)` `tuple`s and
-      returns the same thing with the gradients modified.
+    clip_gradients: float, callable or `None`. If a float is provided, a global
+      clipping is applied to prevent the norm of the gradient from exceeding
+      this value. Alternatively, a callable can be provided, e.g.,
+      adaptive_clipping_fn().  This callable takes a `list` of 
+      `(gradients, variables)` `tuple`s and returns the same thing with the
```
Comment (fchollet) [inline tensorflow/contrib/layers/python/layers/optimizers.py]: No need to use backquotes around "list" and "tuple"

**Review by fchollet — COMMENTED** [review]
Thank you for the PR.

**Comment by schaber** [comment]
@fchollet Thank you for the PR review.  I made changes according to your comments about 20 days ago and thought I left a comment that would alert you, but I am "@-mentioning" you again just in case.

**Review by martinwicke — APPROVED** [review]

**Pull request merged**